tutorials/plugins/gdnative/gdnative-cpp-example.rst

```diff
@@ -135,7 +164,39 @@ and ``src`` directories in your GDNative module.
 In the ``src`` folder, we'll start with creating our header file
 for the GDNative node we'll be creating. We will name it ``gdexample.h``:
 
-.. code:: C++
+.. tabs::
+ .. code-tab:: C++ NativeScript 1.1
+
+    #ifndef GDEXAMPLE_H
+    #define GDEXAMPLE_H
+
+    #include <Godot.hpp>
+    #include <Sprite.hpp>
+
+    namespace godot {
+
+    class gdexample : public Sprite {
```

Comment:
Doesn't matter so much, but since we use `PascalCase` for class names in Godot, it might make sense to use the same style in GDNative examples?

Reply:
as in gd_example?
Sprite is what Sprite is, that's just the bindings library, I didn't chose it

Reply:
No, I mean `GDExample`. I've never seen a `class snake_case` in Godot's codebase.

Reply:
Oh ok